### ISSUE TITLE
Use natural sorting for mapvotes instead of strict alphabetic sorting

### ISSUE THREAD

**Issue by H-M-H** [ddnet/ddnet-django#6]
Replace this line with it then: https://github.com/ddnet/ddnet-django/blob/3b74e252161e1452f248901d85c85c08a1b65f7e/maps/utils.py#L87

**Comment by H-M-H** [comment]
This should work: https://stackoverflow.com/a/4836734

**Comment by heinrich5991** [comment]
Better to do it in the database, I guess: https://stackoverflow.com/a/42535635/870079

**Comment by H-M-H** [comment]
Yes, I would also prefer sorting in the database but the solution proposed implies adding another field to the maps table with all numbers from the mapnames replaced with zeropadded ones. Do we want this ?

**Comment by heinrich5991** [comment]
I don't know, I think you're the expert here. :)
